src/app/query/stacks/microblock/microblock.query.ts

```diff
@@ -6,7 +6,10 @@ import { createMoney } from '@shared/models/money.model';
 import { sumNumbers } from '@app/common/math/helpers';
 import { useStacksClientUnanchored } from '@app/store/common/api-clients.hooks';
 
-export function useCurrentAccountMicroblockBalanceQuery(address: string) {
+export function useCurrentAccountMicroblockBalanceQuery(
+  address: string,
+  pendingTxsBalance?: number
```

Comment:
wonder if there could be some better approach🤔

Reply:
Invalidating the data directly in `useAccountMempoolQuery` the root query that fetches?

either in `onSuccess` or a `useEffect` call,

```
queryClient.invalidateQueries({ queryKey: ['account-microblock', address] }) // you can also just omit address and that will clear them all which is okay I think
```

Reply:
done